ui/docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+services:
+  ui:
+    image: node
+    working_dir: /w
+    volumes:
+      - .:/w:cached
+    command: npm start -- --host=0.0.0.0 ${ENV_FLAG}
+    ports:
+      - 4200:4200
```

Comment:
We have one of these for FireCloud and I never use it. I generally run only the services I'm working on at any given moment. I certainly won't discourage the effort if someone decides this is useful and creates it.